### PR TITLE
Document Redeploy of EFK certificates

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1658003

### CODE REVIEW

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -1974,6 +1976,14 @@ $ curl --key /etc/elasticsearch/secret/admin-key \
...
 ----
 
+[[fluentd-redeploy-certs]]
+== Redeploy EFK Certificates
+
+If you need to redeploy the EFK certificates, first remove the certificates from the `{/location/of/your/base/ocp/install}/logging` directory 
+on your the first master listed in the Ansible host inventory. Then re-run the logging installer. 
+Make sure the xref:logging-custom-certificates[Custom Certificate] parameters are set 
+in your xref:../install/configuring_inventory_file.adoc#configuring-ansible[inventory host file].
+
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: @ewolinetz Do we have a default location for the certificate directory, or something more real-world than `location/of/your/base/ocp/install/logging`?

**Comment by mburke5678** [comment]
@xltian PTAL thank you!

File: install_config/aggregate_logging.adoc (hunk 1 above)
Comment (ewolinetz) [reply install_config/aggregate_logging.adoc]: @mburke5678 do we have a defined location in our docs of where the master/node configs are installed? it is the same base path as that (and it is different for an origin and enterprise install)

**Comment by xltian** [comment]
@anpingli   Please review this doc.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL. I have one question out to QE, but I think we can peer review.

File: install_config/aggregate_logging.adoc (hunk 1 above)
Comment (anpingli) [reply install_config/aggregate_logging.adoc]: @mburke5678 No,  Redeploy the pod is enough.( the pod wasn't restarted as the step 2 failed. so we need to redeploy pod manually)

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL. Not a large change. Most of the added text is code block.

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -1974,6 +1976,51 @@ $ curl --key /etc/elasticsearch/secret/admin-key \
   "https://localhost:9200/project.logging.3b3594fa-2ccd-11e6-acb7-0eb6b35eaee3.2016.06.15"
 ----
 
+[[fluentd-redeploy-certs]]
+== Redeploying EFK Certificates
+
+If you need to redeploy the EFK certificates, first remove the certificates from the `*/etc/origin/logging*` directory
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: Then, re-run

Better yet, you could make these 3 points an ordered list and drop "first" and "then" in the statements.

**Comment by ahardin-rh** [comment]
@mburke5678 Just one minor comment from me. :+1:

**Pull request merged**